src/wp-includes/user.php

```diff
@@ -110,6 +110,26 @@ function wp_signon( $credentials = array(), $secure_cookie = '' ) {
 	}
 
 	wp_set_auth_cookie( $user->ID, $credentials['remember'], $secure_cookie );
+
+	/**
+	 * @global wpdb $wpdb WordPress database abstraction object.
+	 */
+	global $wpdb;
+
+	// Flush `user_activation_key` if exists after successful login.
+	if ( ! empty( $user->user_activation_key ) ) {
+		$wpdb->update(
```

Comment:
Hi @Rajinsharwar 

I would just suggest passing the parameters `$format` and `$where_format` to reinforce the expected formats.

```php
$wpdb->update(
	$wpdb->users,
	array( 'user_activation_key' => '', ),
	array( 'ID' => $user->ID ),
	array( '%s' ),
	array( '%d' )
);
```

Other than that, looks good to me :raised_hands:

Reply:
Thanks for your suggestion! Added :)